### PR TITLE
[review] Improve build experience

### PR DESCRIPTION
**Pull request by dawagner**
@krocard @tcahuzax Please review
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/krocard%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/290%23issuecomment-149296957%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222015-10-19T17%3A55%3A16Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20made%20a%20small%20change%20to%20the%20last%20commit%2C%20on%20which%20we%20orally%20agreed%20upon.%22%2C%20%22created_at%22%3A%20%222015-10-19T18%3A32%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20c6839a7d38ea5fc45d824c26fdfcacc09617c349%20README.md%2043%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/290%23discussion_r42264312%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Why%20so%20%3F%20I%20have%20never%20used%20absolute%20patch%20for%20CMake%20options%20%3F%22%2C%20%22created_at%22%3A%20%222015-10-16T16%3A56%3A32Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22Seen%20with%20you%3A%20it%20is%20necessary%20-%20at%20least%20on%20Windows.%20I%20can%20add%20an%20explanation%20if%20you%20like%3A%20%5C%22You%20don%27t%20know%20where%20relative%20paths%20will%20be%20evaluated%20from%5C%22.%5Cr%5Cn%5Cr%5CnEDIT%3A%20%3A%2B1%3A%20done%22%2C%20%22created_at%22%3A%20%222015-10-19T14%3A31%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%2C%20%7B%22body%22%3A%20%22Never%20mind%2C%20keep%20it%20like%20this.%20%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-10-19T14%3A52%3A34Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20README.md%3AL64-110%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/01org/parameter-framework/pull/290%23issuecomment-149296957%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/290%23discussion_r42264312%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/290%23discussion_r42377726%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/290%23discussion_r42380477%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/290%23issuecomment-149306373%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/krocard'></a>

- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/290#issuecomment-149296957'>General Comment</a></b>
- <a href='https://github.com/krocard'></a> :shipit:
- <a href='https://github.com/dawagner'></a> I made a small change to the last commit, on which we orally agreed upon.
- [x] <a href='#crh-comment-Pull c6839a7d38ea5fc45d824c26fdfcacc09617c349 README.md 43'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/290#discussion_r42264312'>File: README.md:L64-110</a></b>
- <a href='https://github.com/krocard'></a> Why so ? I have never used absolute patch for CMake options ?
- <a href='https://github.com/dawagner'></a> Seen with you: it is necessary - at least on Windows. I can add an explanation if you like: "You don't know where relative paths will be evaluated from".
EDIT: :+1: done
- <a href='https://github.com/krocard'></a> Never mind, keep it like this. :+1:

<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/290?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/290?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/290?approve=0'></a>&nbsp;<a href='https://github.com/01org/parameter-framework/pull/290'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dawagner** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/01org/parameter-framework/pull/290'>Refresh</a>.<a href='#crh-update-none'></a>

File: README.md [hunk 1]
```diff
@@ -64,11 +64,46 @@ See [the wiki on github](https://github.com/01org/parameter-framework/wiki).
...
+### How-To
+
+If you are already familiar with CMake, you know what to do.
+
 Run `cmake .` then `make`.  You may then install libraries, headers and
 binaries with `make install`.  By default, they are installed under
 `/usr/local` on unix OSes; if you want to install them under a custom
 directory, you may do so by passing it to the `cmake .` command; e.g.
 
+    # Always use absolute paths in CMake options
```
Comment (krocard) [inline README.md]: Why so ? I have never used absolute patch for CMake options ?

Comment (dawagner) [inline README.md]: Seen with you: it is necessary - at least on Windows. I can add an explanation if you like: "You don't know where relative paths will be evaluated from".

Comment (krocard) [inline README.md]: Never mind, keep it like this. :+1
---------------------------------------------------------------------
Intel Corporation SAS (French simplified joint stock company)
Registered headquarters: "Les Montalets"- 2, rue de Paris, 
92196 Meudon Cedex, France
Registration Number:  302 456 199 R.C.S. NANTERRE
Capital: 4,572,000 Euros

This e-mail and any attachments may contain confidential material for
the sole use of the intended recipient(s). Any review or distribution
by others is strictly prohibited. If you are not the intended
recipient, please contact the sender and delete all copies.

**Comment by krocard** [comment]
:shipit:

**Comment by dawagner** [comment]
I made a small change to the last commit, on which we orally agreed upon.

**Pull request merged**